### ISSUE TITLE
Support docker-compose v2 

### ISSUE THREAD

**Issue by distorhead** [werf/werf#5010]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Problem

Error occurs when 
```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the services key, or omit the version key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

### Solution (if you have one)

Werf runs `docker-compose up` to run compose, but in v2 we use `docker-compose-plugin` which runs with `docker compose up` command.

So we need to detect compose version and if version >= 3.8 werf should call `docker compose up` instead of `docker-compose up`.

### Additional information

As workaround create wrapper which calls docker compose for example in `/usr/local/bin/docker-compose`:

```
#!/bin/bash
exec docker compose "$@"
```

**Comment by ilya-lesikov** [comment]
Since v1.2.283 we use `docker compose`.

**Comment by shurup** [comment]
Hi @ilya-lesikov! I think I just had a similar issue while building `werf/website` following [these instructions](https://github.com/werf/website/blob/main/DEVELOPMENT.md) with werf v2.3.3. Running `task compose:up` resulted in:

```
Error: docker-compose: exec: "docker-compose": executable file not found in $PATH
```

I had `docker-compose-v2` installed in Ubuntu, but no `docker-compose` installed. Using this workaround (creating a simple `/usr/local/bin/docker-compose` wrapper to `exec docker compose "$@"`) helped.

**Comment by ilya-lesikov** [comment]
@shurup Fixed in #6207

**Comment by shurup** [comment]
Great, thank you!
